orchestra/models.py

```diff
@@ -461,6 +461,43 @@ def __str__(self):
             str(self.task), self.assignment_counter, str(self.worker))
 
 
+class Iteration(models.Model):
+    """
+    Iterations are the contiguous units of a worker's time on task.
+
+    Attributes:
+        start_datetime (datetime.datetime):
+            The time the task was assigned for this iteration.
+        end_datetime (datetime.datetime):
+            The time the iteration was completed. Will be None if the iteration
+            is currently in progress.
+        assignment (orchestra.models.TaskAssigment):
+            The task assignment to which the iteration belongs.
+        action (orchestra.models.Iteration.Action):
```

Comment:
idea:  if we made it non-nullable, we could call it a `status` or `state`, which could be one of `PROCESSING, SUBMITTED, ACCEPTED, REJECTED`